docs/source/tutorials/plots_intro.ipynb

```diff
@@ -8,10 +8,9 @@
     "(plots_intro)=\n",
```

Comment:
<p>Re overlay, it would work the same if we used "overlay_school" instead. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz-plots/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>